scripts/las2grain.py

```diff
@@ -24,8 +24,8 @@ def calc_transmittance(altitude_km,
                        lrt_dir='/Users/brent/Code/ice-road-copters/test', 
                        path_to_libradtran_base='/Users/brent/Documents/Albedo/libRadtran-2.0.4/',
                        atmos='mw',
-                       h=5.93 ,
-                       aod=0.095):
+                       h=4.79 ,
+                       aod=0.073):
```

Comment:
Minor: Some references and documentation to these number would be nice

Reply:
@jomey referenced in the next line. 

    NOTE: h - H20 vapor in mm from AERONET
          aod - is aerosol optical depth at 870 nm from AERONET 
          altitude_km is the altitude of the AERONET station in Merdian, ID
          atmos - assuming mid latitude winter

          https://aeronet.gsfc.nasa.gov/

    Both are from the Meridian_DEQ site. For now I have been updating h and aod variables for each date.
    

    On the physics, the attenuation is relatively small using our methodology, and only really shows a small 1% 
    change when the range (or beam distance) varies largely from the calibration range at Eagle.

    This function is used in a for loop to solve for extinction coef based on change in altitude and transmittance.